index.html

```diff
@@ -566,6 +571,117 @@ <h1>Trust Model</h1>
 
     <section>
       <h1>Basic Concepts</h1>
+
+      <section>
+        <h2>Types</h2>
+        <p>
+Type information may be expressed via the <a>type</a> property:
+        </p>
+        <dl>
+          <dt><dfn>type</dfn></dt>
+          <dd>
+The value of this property MUST be one or more URIs.
+If more than one URI is provided, the array MUST be interpreted as a set of
+values where order is not significant. Note that syntax such as JSON-LD terms
+MAY be used to ease developer usage. It is RECOMMENDED that dereferencing the
+URI results in a document containing machine-readable information about the
+type.
+          </dd>
+        </dl>
+
+        <pre class="example nohighlight" title="Usage of the type property">
+{
+  "id": "http://dmv.example.gov/credentials/3732",
+  <span class="highlight">"type": ["VerifiableCredential", "ProofOfAgeCredential"]</span>,
+  "claim": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "ageOver": 21
+  },
+  "proof": { ... }
+}
+        </pre>
+
+        <p>
+With respect to this specification, the following <a>type</a>s MUST be
+specified on the following objects:
+        </p>
+
+        <table class="simple">
+          <thead>
+            <tr>
+              <th>Concept</th>
+              <th>Type</th>
+            </tr>
+          </thead>
+
+          <tbody>
+            <tr>
+              <td>
+A <a href="#credentials">credential</a> object
+              </td>
+              <td>
+<tt>"type": ["<code>VerifiableCredential</code>"]</tt>
+              </td>
+            </tr>
+
```

Comment:
Fixed in 1bd1fd5eb634ea0082956e19268b3c33420b0ace.

I couldn't add it to VerifiablePresentation because there are no other types at present that are contemplated.

Reply:
Yes fine, thanks